diff --git a/platform/chromium/vapi-background.js b/platform/chromium/vapi-background.js
index 306b41b..4246719 100644
--- a/platform/chromium/vapi-background.js
+++ b/platform/chromium/vapi-background.js
@@ -55,10 +55,12 @@ if (
 
 // https://issues.adblockplus.org/ticket/5695
 // - Good idea, adopted: cleaner way to detect user-stylesheet support.
-vAPI.supportsUserStylesheets = 
+vAPI.supportsUserStylesheets = vAPI.chromiumVersion >= 66;
+/*
     chrome.extensionTypes instanceof Object &&
     chrome.extensionTypes.CSSOrigin instanceof Object &&
     'USER' in chrome.extensionTypes.CSSOrigin;
+*/
 vAPI.insertCSS = chrome.tabs.insertCSS;
 
 var noopFunc = function(){};
